### ISSUE TITLE
save data corupted

### ISSUE THREAD

**Issue by younesko86** [soulsmods/ModEngine2#130]
when i put normale game regulations bin it runs normlay but when i put the mod regulations bin i get this message save data corupted any fix???(url)

**Comment by erza2209** [comment]
You find a solution?

**Comment by younesko86** [comment]
> You find a solution?

Yes you need to update the to latest version 1.9.1 that will fix it
